cheminformatics/deploy/workflows/Cheminformatics-4-TransFS_scoring.ga

```diff
@@ -0,0 +1,302 @@
+{
```

Comment:
@bedroesb there are wokflows in each sub-folder, can you remove them? It does not make sense to have them in two places.

Reply:
I will compare them to make sure I have the right ones and than delete them